### DISCUSSION TITLE
Creating Anki flashcards decks from the vocabulary in this repo

### DISCUSSION THREAD

**Discussion by Ronserruya** [SethClydesdale/genki-study-resources#88] [Ideas]
Hello, I very recently started learning Japanese with Genki, I saw that almost everywhere online people recommend using flashcards and SRS to learn vocabulary, but I saw in https://ankiweb.net/shared/decks/genki that there are no decks for the new 3rd edition.

I found your site and while I think it's incredible and I'm definitely going to continue using it, I'd like to also use Anki as well.
I wrote a short script that "scrapes" this repo to grab all of the vocabularies and create Anki decks from it that are tagged with the Genki lesson, type, and topics.

I wanted to know if you'd like me to commit this script/the Anki decks it creates :)

Ps. While I used it to create decks for the 3rd edition, the script also works on all of the Genki 2 files (apart from lesson 14, file 6)
Ps2. Your style of code writing is very consistent across the entire repo so it was a pleasure to work with 👍 

Thank you for all the work you put into this site :)

**Comment by SethClydesdale** [comment]
Hello @Ronserruya,

Sure, that'd be wonderful if you'd like to commit the script! I actually have [a folder](https://github.com/SethClydesdale/genki-study-resources/tree/master/resources/tools) in the repo with a few tools for working on the site, so I think it'd fit nicely in there. Of course so people can find and use it, putting info in the readme -- probably under or after the "Resources for Studying Japanese" section -- that leads to it would be a good idea. 😉

You're welcome, and thanks as well! 👍
